File: from-3501-to-3600/3516-find-closest-person/solution.go
func findClosest(x int, y int, z int) int {
    d1, d2 := z - x, z - y
    if d1 < 0 { d1 = -d1 }
    if d2 < 0 { d2 = -d2 }
    if d1 < d2 {
        return 1
    } else if d1 > d2 {
        return 2
    }
    return 0
}

File: from-3501-to-3600/3516-find-closest-person/solution.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        d1, d2 = abs(z - x), abs(z - y)

        if d1 < d2:
            return 1
        elif d1 > d2:
            return 2

        return 0